specification/hugr.md

```diff
@@ -1821,7 +1847,13 @@ These operations allow this.
     in first order graphs as straightforward (albeit expensive)
     manipulations of Graph `struct`s/protobufs\!
 
-<img src="attachments/2647818241/2647818326.png" height="64px">
+$\displaystyle{
```

Comment:
This equation isn't rendering. I can't see anything obviously wrong, but it's the only one making use of newlines within latex - maybe github doesn't like that?

Reply:
Ah hadn't spotted that, let me try and fix it.

Reply:
Your guess was right. Made it a single line; seems OK now.